src/harness/fourslash.ts

```diff
@@ -2846,17 +2846,17 @@ Actual: ${stringify(fullActual)}`);
             }
         }
 
-        public applyRefactor({ refactorName, actionName, actionDescription, newContent: newContentWithRenameMarker }: FourSlashInterface.ApplyRefactorOptions) {
+        public applyRefactor({ refactorName, description, actionName, actionDescription, newContent: newContentWithRenameMarker }: FourSlashInterface.ApplyRefactorOptions) {
             const range = this.getSelection();
             const refactors = this.languageService.getApplicableRefactors(this.activeFile.fileName, range);
-            const refactor = refactors.find(r => r.name === refactorName);
+            const refactor = refactors.find(r => r.name === refactorName && description === undefined || description === r.description);

```

Comment:
I don't have a detailed knowledge of precedence in JS, but I would have guessed you needed parens for the `||`.